internal/pkg/cli/deploy/workload.go

```diff
@@ -110,12 +110,13 @@ type fileReader interface {
 
 // StackRuntimeConfiguration contains runtime configuration for a workload CloudFormation stack.
 type StackRuntimeConfiguration struct {
```

Comment:
nit: is there a way of separating these fields between "required" and "optional"? Like which one of these are guaranteed to be populated vs. depending on the circumstance.

For example,
```go
// Required data that is populated on every package.
ImageDigests
CustomResourceURLs

// Optional data populated depending on customer's input.
EnvFileARNs..
```

Reply:
i'm not _opposed_ to doing this, however, i think the only field that's actually required across all workloads is `CustomResourceURLs`...